manifests/generated/operator-csv.yaml.in

```diff
@@ -754,6 +754,15 @@ spec:
           - create
           - get
           - delete
+        - apiGroups:
```

Comment:
Please amend

Reply:
I am not sure what you mean here? It is in a separate commit https://github.com/kubevirt/kubevirt/pull/14365/commits/2c73a7123bd030b5bbebf0c12104a8bbc93e7457 right after adding dra controller